src/Menu/Menu.js

```diff
@@ -381,14 +402,14 @@ class Menu extends Component {
     this.props.onItemTouchTap(event, item, index);
   }
 
-  incrementKeyboardFocusIndex(filteredChildren) {
+  incrementKeyboardFocusIndex(event, filteredChildren) {
     let index = this.state.focusIndex;
     const maxIndex = this.getMenuItemCount(filteredChildren) - 1;
 
     index++;
     if (index > maxIndex) index = maxIndex;
 
-    this.setFocusIndex(index, true);
+    this.setFocusIndex(index, true, event);
```

Comment:
We use the `event` as the last and first argument. I think that it should always been the first argument.
Could we change that here?